### ISSUE TITLE
Multi-Monitor throw window to space on that monitor

### ISSUE THREAD

**Issue by hpfmn** [ianyh/Amethyst#1311]
Since 0.16 the behavior of throwing a window to a different space has changed. Before it would be put to the "relative" number of that space on that monitor (e.g. if your current monitor starts with space 10 - throwing it on 1 would put it in space 10) but now it uses "absolute" spacing - so if you put it on space 1 it gets thrown on the monitor that has space 1.

This might be feature for some people but I'd like to have the old behavior and maybe a setting to change it.

**Comment by artmazi** [comment]
Yep, I have the same issue and couldn't find a workaround (other than using the mouse).

**Comment by TLenfers** [comment]
I can confirm the issue. The only workaround I found was using an additional program. But the mouse will not follow the window.

**Comment by 27medkamal** [comment]
I can confirm the same issue
